models/transformation/diagnosis_pivot.sql

```diff
@@ -1,145 +1,105 @@
-with diagnois_pivot as(
-  select 
-      *
-  from 
-      (select 
+with diagnois_pivot as (
+
+    select
           cur_clm_uniq_id
-          ,bene_mbi_id
-          ,clm_dgns_cd
-          ,clm_val_sqnc_num
-          ,dgns_prcdr_icd_ind
-       from {{ var('parta_diagnosis_code')}}
-       )
-  pivot(
-      max(clm_dgns_cd) for clm_val_sqnc_num in ('1'
-                                              , '2'
-                                              , '3'
-                                              , '4'
-                                              , '5'
-                                              , '6'
-                                              , '7'
-                                              , '8'
-                                              , '9'
-                                              , '10' 
-                                              , '11' 
-                                              , '12' 
-                                              , '13' 
-                                              , '14' 
-                                              , '15' 
-                                              , '16' 
-                                              , '17' 
-                                              , '18' 
-                                              , '19' 
-                                              , '20' 
-                                              , '21' 
-                                              , '22' 
-                                              , '23' 
-                                              , '24' 
-                                              , '25' 
-                                              )
-    )piv (cur_clm_uniq_id, bene_mbi_id, dgns_prcdr_icd_ind, diagnosis_code_1, diagnosis_code_2, diagnosis_code_3, diagnosis_code_4, diagnosis_code_5, diagnosis_code_6
-         , diagnosis_code_7, diagnosis_code_8, diagnosis_code_9, diagnosis_code_10, diagnosis_code_11, diagnosis_code_12, diagnosis_code_13, diagnosis_code_14, diagnosis_code_15
-         , diagnosis_code_16, diagnosis_code_17, diagnosis_code_18, diagnosis_code_19, diagnosis_code_20, diagnosis_code_21, diagnosis_code_22, diagnosis_code_23, diagnosis_code_24
-         , diagnosis_code_25)
-)
-, poa_pivot as(
-  select 
-  *
-  from 
-      (select 
+        , bene_mbi_id
+        , dgns_prcdr_icd_ind
+        , {{ dbt_utils.pivot(

```

Comment:
Using the pivot macro from dbt utils since the syntax was different for BigQuery.